x-pack/plugins/maps/public/classes/styles/vector/vector_style.tsx

```diff
@@ -458,7 +467,10 @@ export class VectorStyle implements IVectorStyle {
       : (this._lineWidthStyleProperty as StaticSizeProperty).getOptions().size !== 0;
   }
 
-  renderEditor(onStyleDescriptorChange: (styleDescriptor: StyleDescriptor) => void) {
+  renderEditor(
+    onStyleDescriptorChange: (styleDescriptor: StyleDescriptor) => void,
+    onCustomIconsChange: (customIcons: Record<string, CustomIcon>) => void
```

Comment:
Type as `onCustomIconsChange: (customIcons: CustomIcon[]) => void`. leaking that this class converts CustomIcon[] to {} for easy lookup should not escape this class